src/pages-and-resources/discussions/DiscussionAppCard.jsx

```diff
@@ -9,8 +9,8 @@ import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
 
 import messages from './messages';
 
-function DiscussionToolOption({
-  intl, forum, selected, onSelect,
+function DiscussionAppCard({
+  intl, app, selected, clickHandler,
```

Comment:
The "onSelect" -> "clickHandler" thing is just using a naming scheme I'm more familiar with; it's functionally equivalent.